eth_account/messages.py

```diff
@@ -187,13 +187,13 @@ def encode_defunct(
         SignableMessage(version=b'E', header=b'thereum Signed Message:\n6', body=b'I\xe2\x99\xa5SF')
 
         # these four also produce the same hash:
-        >>> encode_defunct(w3.toBytes(text=message_text))
+        >>> encode_defunct(w3.toBytes(text=message_text)) # doctest: +SKIP
```

Comment:
Just curious what the failure was here (I'm fine to push it to another issue)

Reply:
```
NameError: name 'w3' is not defined
```

Reply:
Hah, ok. Well that's not an instance method so it should be straightforward to switch `w3` to `Web3` and just add a `from web3 import Web3` up top. Just a note for when you/someone does the follow-up work.